Common/src/Pollster/RequestProcessor.cs

```diff
@@ -333,30 +334,29 @@ await output.WhenAll()
                                                                  singleReplyStream,
                                                                  cancellationToken)
                                        .ConfigureAwait(false);
-          await workerStreamHandler_.WorkerRequestStream.WriteAsync(new ProcessRequest
+          await workerStreamHandler_.Pipe.WriteAsync(new ProcessRequest
+                                                     {
+                                                       CreateTask = new ProcessRequest.Types.CreateTask
                                                                     {
-                                                                      CreateTask = new ProcessRequest.Types.CreateTask
-                                                                                   {
-                                                                                     Reply   = replyLargeTasksAsync,
-                                                                                     ReplyId = first.RequestId,
-                                                                                   },
+                                                                      Reply   = replyLargeTasksAsync,
+                                                                      ReplyId = first.RequestId,
                                                                     },
-                                                                    CancellationToken.None)
```

Comment:
No more CancellationToken for this switch case ?

Reply:
We made an interface above this that wrap up a grpc type that has this cancellation token in its implementation (grpcAsyncPipe). It has a None cancellation token because its implementation does not support a cancellation token.